src/pages/list/basic-list/index.tsx

```diff
@@ -0,0 +1,378 @@
+import {
+  Avatar,
+  Button,
+  Card,
+  Col,
+  DatePicker,
+  Dropdown,
+  Input,
+  List,
+  Menu,
+  Modal,
+  Progress,
+  Radio,
+  Row,
+  Select,
+  Result,
+} from 'antd';
+import React, { Component } from 'react';
+
+import { Dispatch } from 'redux';
+import { FormComponentProps } from 'antd/es/form';
+import { Form } from '@ant-design/compatible';
+import { PlusOutlined, DownloadOutlined } from '@ant-design/icons';
+import { PageHeaderWrapper } from '@ant-design/pro-layout';
+import { connect } from 'dva';
+import { findDOMNode } from 'react-dom';
+import moment from 'moment';
+import { StateType } from './model';
+import { BasicListItemDataType } from './data.d';
+import styles from './style.less';
+
+const FormItem = Form.Item;
+const RadioButton = Radio.Button;
+const RadioGroup = Radio.Group;
+const SelectOption = Select.Option;
+const { Search, TextArea } = Input;
+
+interface BasicListProps extends FormComponentProps {
+  listAndbasicList: StateType;
+  dispatch: Dispatch<any>;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=4583897)